### PR TITLE
Update configure-dynamic-emergency-calling.md

### PR DESCRIPTION
**Pull request by nitinjaitely**
Support for web has been released.

### CODE REVIEW

File: Teams/configure-dynamic-emergency-calling.md [hunk 1]
```diff
@@ -76,11 +76,13 @@ The following clients are currently supported.  Check back often to see updates
 
 - Teams desktop client for Microsoft Windows
 - Teams desktop client for Apple macOS
+- Teams Web client
```
Comment (JohanFreelancer9) [inline Teams/configure-dynamic-emergency-calling.md:79]:
```suggestion
- Teams web client
```

File: Teams/configure-dynamic-emergency-calling.md [hunk 2]
```diff
@@ -90,8 +92,6 @@ The following clients are currently supported.  Check back often to see updates
 > - Mac OS, which requires [LLDP enablement software](https://www.microsoft.com/download/details.aspx?id=103383).
 > - Teams phone with Teams app version 1449/1.0.94.2021110101 and later.
 
-> [!NOTE]
-> Dynamic emergency calling, including security desk notification, isn't supported on the Teams web client. To prevent users from using the Teams web client to call PSTN numbers, you can set a Teams calling policy and turn off the **Allow web PSTN calling** setting. To learn more, see [Calling policies in Teams](teams-calling-policy.md) and [Set-CsTeamsCallingPolicy](/powershell/module/skype/set-csteamscallingpolicy).
```
Comment (JohanFreelancer9) [inline Teams/configure-dynamic-emergency-calling.md:95]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nitinjaitely, kindly commit the minor suggested changes. Many thanks.

**Comment by CarolynRowe** [comment]
According to @roykuntz , web client is still not supported--except for some ability for the user to enter an address.  Will follow up with Roy about addressing the latter, but closing this PR so that dot get merged erroneously.

**Pull request closed without merging**